x-pack/plugin/stack/qa/rest/src/test/resources/rest-api-spec/test/stack/10_basic.yml

```diff
@@ -0,0 +1,105 @@
+---
+setup:
+  - do:
+      cluster.health:
+        wait_for_status: yellow
+
+---
+"Test stack template installation":
+  - do:
+      ilm.get_lifecycle:
+        policy: "logs"
```

Comment:
will these fail when missing? Should we match the status code?

Reply:
Yeah, these definitely fail when the policy/template/etc is missing